modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -515,18 +515,20 @@ public CPDefinition copyCPDefinition(long cpDefinitionId)
 			cpDefinitionId);
 
 		return cpDefinitionLocalService.copyCPDefinition(
-			cpDefinitionId, cpDefinition.getGroupId());
+			cpDefinitionId, cpDefinition.getGroupId(),
+			cpDefinition.getUserId());
```

Comment:
@MahmoudAzzam @alessiorendina confirmed my doubt, so please move userid to be first param here and make baseline run